### PR TITLE
New Language Support for Rust, Golang, Ruby

### PR DESCRIPTION
**Pull request by PhoenixFlame101**
With reference to #2.

Added code that can remove `print` and `log` statements (or their analogues) from the languages:
- Rust
- Golang
- Ruby

### CODE REVIEW

**Comment by ArnavKumar7** [comment]
!bounty 50

File: src/constants.js [hunk 1]
```diff
@@ -2,23 +2,36 @@ const langs = {
...
 };
 
 const patterns = {
   py: {
     print: "^[ \t]*print[ \t]*\(.*\)",
     printComment: "^[ \t]*#[ \t]*print[ \t]*\(.*\)"
   },
+  rb: {
+    puts: "^[ \t]*puts[ \t]*$",
+    putsComment: "^[ \t]*#[ \t]*puts[ \t]*$"
```
Comment (avinash-vk) [inline src/constants.js:19]: Hey, some slight things to note: The dollar sign here, doesn't trigger the patterns and I'd suggest replacing it with `(.*\)`.

File: src/extension.ts [hunk 2]
```diff
@@ -66,6 +66,66 @@ export function activate(context: vscode.ExtensionContext) {
...
+        for (var i = 0; i < textEditor.document.lineCount; i++) {
+          let linetext = textEditor.document.lineAt(i).text;
+          var newline = linetext;
+
+          if (linetext.match(patterns.rs.println)) {
+            var j = 0;
+            for (j = 0; j < linetext.length; j++) {
+              if (linetext[j] != ' ' && linetext[j] != '\t') break;
+            }
+            newline = [linetext.slice(0, j), "# ", linetext.slice(j)].join('');
```
Comment (avinash-vk) [inline src/extension.ts:82]: Rust comments are "//" and not "#"

**Review by avinash-vk — CHANGES_REQUESTED** [review]
Changes mentioned above, do test it with the scripts to verify all edge cases. 

Go also has a `fmt.print` statement you'd have to consider.
